packages/react-reconciler/src/ReactFiberContext.js

```diff
@@ -28,14 +27,19 @@ if (__DEV__) {
   warnedAboutMissingGetChildContext = {};
 }
 
+export const emptyContextObject = {};
```

Comment:
This is exported because there are subtle assumptions around its referential equality in a few places throughout the Fiber codebase. It's used a sentinel value. This makes it explicit for this particular use case.